src/api.jl

```diff
@@ -0,0 +1,188 @@
+"""
+    Julja2ParsedTemplate
+
+Represents a parsed template with executable generator code.
+
+## Fields:
+- `generator_code::Expr`: The expression that generates the code for the template.
+"""
+struct Julja2ParsedTemplate
+generator_code::Expr
+end
+
+"""
+    BlockBorders
+
+This structure contains borders of block.
+
+## Fields:
+- `left::String`: The left border string.
+- `left_length::Int`: The length of the left border.
+- `right::String`: The right border string.
+- `right_length::Int`: The length of the right border.
```

Comment:
No, but it is move convenient to see in code:
`
i += options.var.left_length
`

rather than:
`
i += length(transcode(UInt8, options.var.left)) 
`
Plus length calculated in constructor.